androidTest/java/com/owncloud/android/authentication/AuthenticatorActivityTest.java

```diff
@@ -162,12 +162,12 @@ public void test1_check_certif_not_secure_no_accept()
 
             // Check that login button is disabled
```

Comment:
Update this comment? `//Check that login button is not displayed`

Reply:
Done!